spec/mailers/request_mailer_spec.rb

```diff
@@ -16,15 +16,10 @@
       receive_incoming_mail('incoming-request-plain.email', ir.incoming_email)
       expect(ir.incoming_messages.size).to eq(2) # one more arrives
       expect(ir.info_request_events[-1].incoming_message_id).not_to be_nil
-
-      deliveries = ActionMailer::Base.deliveries
-      expect(deliveries.size).to eq(1)
-      mail = deliveries[0]
-      expect(mail.to).to eq([ 'bob@localhost' ]) # to the user who sent fancy_dog_request
-      deliveries.clear
```

Comment:
I went through them all and in every case the tests were either expecting the mailbox to be empty or contain one request - I can reinstate the former expectation if you think it's going to be a problem, as at least that way we'd see when something bad happens, but I don't *think* these changes should cause a problem.